.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: bash
+services: docker
+script:
+- "./scripts/cibuild"
+deploy:
```

Comment:
This appears to be uploading all of the source files vs. just the API spec.

```bash
$ aws --profile raster-foundry s3 ls s3://rasterfoundry-staging-docs-site-us-east-1/spec/
                           PRE scripts/
2018-06-26 13:57:23      11341 LICENSE
2018-06-26 13:57:23         26 README.md
2018-06-26 13:57:23        162 docker-compose.yml
2018-06-26 13:57:23        266 package.json
2018-06-26 14:03:58     112687 spec.yml
2018-06-26 13:57:23      15470 yarn.lock
```

Reply:
I forgot to resolve that.